sampling/sample_creator.py

```diff
@@ -1,20 +1,45 @@
-class SampleCreator():
+from multiprocessing import Process
```

Comment:
Let's stick with torch.multiprocessing? As we discussed earlier, it allows for variable(s) sharing in the heap space with ease, and was the reason we chose it in the first place.